tests/testthat/test-utilities-simulation-results.R

```diff
@@ -166,12 +166,12 @@ test_that("simulationResultsToDataFrame works as expected - minimal pkml", {
   expect_equal(unique(df2$paths), "Organism|A")
 
   # names
-  expect_equal(
-    names(df1),
-    c(
-      "paths", "IndividualId", "Time", "simulationValues", "unit",
-      "dimension", "TimeUnit", "TimeDimension", "molWeight"
-    )
+  expect_true(
```

Comment:
Changed the test so it does not test for the exact order of columns. The order has changed to 

`"IndividualId"     "Time"             "paths"            "simulationValues" "TimeDimension"    "TimeUnit"         "dimension"        "unit"             "molWeight"  `

Reply:
This tests that there are no additional columns in `names(df1)`, but not that all columns are present. Maybe 
```
expect_true(all(names(df1) %in% c(...))
expect_true(all(c(...) %in% names(df1))
```
or
```
expect_equal(sort(names(df)), sort(c(...)))
```